README.md

```diff
@@ -24,9 +24,17 @@ git clone https://github.com/Project-Summer-AI-Lab-Glider/nurse-scheduling-probl
 
 ```bash
 cd nurse-scheduling-problem-solver
+```
+3. Install dependecies
+```
+$ julia 
```

Comment:
jak dajesz tutaj znak prompta, to przyadlby się w innych częściach instrukcji

Reply:
Też masz rację. Zupdatuję też inne miejsca